### PR TITLE
Add use cases to DG add cmd & add parent

### LINKED ISSUE #5: Update the DG: use cases

### PR DESCRIPTION
**Pull request by Ongzl**
Resolves #5

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -308,7 +308,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use case: Add a person**
+
+**MSS**
+1. User adds details of new person to be added to AddressBook.
+2. AddressBook adds new person with relevant details.
```
Comment (softmagnet) [inline docs/DeveloperGuide.md:315]: Should you still be using 'AddressBook though'

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -308,7 +308,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+    Use case ends.
+
+**Extension**
+* 1a. User keys in add command with invalid format.
+    *1a1. AddressBook shows an error message.
```
Comment (softmagnet) [inline docs/DeveloperGuide.md:321]: same here

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -308,7 +308,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+  
+    Use case resumes at step 1.
+
+**Use case: Add parent contact**
+
+**MSS**
+1. User adds detail of parent contact to existing contact.
+2. AddressBook updates the parent contact field with the added details.
```
Comment (softmagnet) [inline docs/DeveloperGuide.md:329]: same here

**Review by softmagnet — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Ongzl) [reply docs/DeveloperGuide.md]: changed to TimeTable

**Pull request merged**